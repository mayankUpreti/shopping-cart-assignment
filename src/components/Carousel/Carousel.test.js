import React  from 'react'
import CustomCarousel from './Carousel'
import {shallow} from 'enzyme';



it('testing a custom carousel ',()=>{
   const mockDb=[{
       id:1212,
       bannerImageUrl:'',
       bannerImageAlt:'yoyo'
   }]
    expect(shallow(<CustomCarousel data={mockDb}/>)).toMatchSnapshot()

})