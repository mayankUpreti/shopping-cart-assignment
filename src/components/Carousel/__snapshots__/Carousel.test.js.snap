// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing a custom carousel  1`] = `
<Carousel
  controls={true}
  defaultActiveIndex={0}
  fade={false}
  indicators={true}
  interval={5000}
  keyboard={true}
  nextIcon={
    <span
      aria-hidden="true"
      className="carousel-control-next-icon"
    />
  }
  nextLabel="Next"
  pause="hover"
  prevIcon={
    <span
      aria-hidden="true"
      className="carousel-control-prev-icon"
    />
  }
  prevLabel="Previous"
  slide={true}
  touch={true}
  wrap={true}
>
  <CarouselItem
    key="1212"
  >
    <img
      alt="yoyo"
      aria-label="yoyo"
      className="d-block w-100"
      src=""
      tabIndex={0}
    />
  </CarouselItem>
</Carousel>
`;
